src/services/redirectToDestination.js

```diff
@@ -35,12 +36,24 @@ export function redirectToDestination(
     }
     case "quiz": {
       redirectURL = import.meta.env.VITE_APP_BASE_URL_QUIZ;
+      const apiKey = import.meta.env.VITE_APP_QUIZ_AF_API_KEY;
+      const unique_id = userID;
```

Comment:
Don't rename the variable. It's called `userId`, let it be the same.

Reply:
sure